packages/runtime/datastore/src/localChannelContext.ts

```diff
@@ -124,23 +123,26 @@ export class LocalChannelContext implements IChannelContext {
      * @param fullTree - true to bypass optimizations and force a full summary tree
      * @param trackState - This tells whether we should track state from this summary.
      */
-    public async summarize(fullTree: boolean = false, trackState: boolean = false): Promise<ISummarizeResult> {
+    public async summarize(fullTree: boolean = false, trackState: boolean = false): Promise<IContextSummarizeResult> {
         assert(this.isLoaded && this.channel !== undefined, "Channel should be loaded to take summary");
         const snapshot = snapshotChannel(this.channel);
         const summary = convertToSummaryTree(snapshot, fullTree);
-        return summary;
+        return {
+            ...summary,
+            references: [],
```

Comment:
This is temporary until #3975 is merged. Then the actual references returned by DDS will be used.